### ISSUE TITLE
Feature Request: Mark each individual track.

### ISSUE THREAD

**Issue by CGraabaek** [erik/derive#38]
Hi there, 

First of all, awesome! 
I was looking to build something like this myself, but then found your awesome project. 

My idea i was thinking about was adding some sort of selector, so you could hightlight each track and see  data like max speed, length and date. 

Here i added a mockup:
![Screenshot 2020-04-22 at 11 27 34](https://user-images.githubusercontent.com/6660378/79965479-5be14800-848c-11ea-9b7e-da1d191b3a97.png)

**Comment by erik** [comment]
Love it! This would definitely be a great addition.

The UI code is pretty ad-hoc (there's no real framework, and a lot of `innerHtml = ...`) , so adding something like you have in the mock might be challenging without some unrelated refactoring. Probably the easiest implementation would be to lean on Leaflet to render popups when clicking on a track, but that doesn't give you the kind of "selector" experience you described.

**Comment by CGraabaek** [comment]
Yeah i agree with you on that part! I just thought it would be a cool addition. 
I liked your idea a lot and am trying to recreate it in VueJS right now. 

So well see if I can get this thing here implemented myself - but awesome project!
